Unable to start reaction

Hi,
when i did `reaction init,` i had a successful Reaction message however some error were present as well.
These errors can also be reproduced by also doing `meteor npm i`:

```
C:\Users\Sylvain\Documents\App\Projects\ecommerce\reaction\reaction>meteor npm i
npm ERR! git clone C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-90f8e8d5 C:\Users\Sylvain\AppData\Local\Temp\npm-12048-ff3f2333\git-cache-30786998\9c5a29ff037c28af07479eaa27591f1b4a0e5b66: Cloning into 'C:\Users\Sylvain\AppData\Local\Temp\npm-12048-ff3f2333\git-cache-30786998\9c5a29ff037c28af07479eaa27591f1b4a0e5b66'...
npm ERR! git clone C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-90f8e8d5 C:\Users\Sylvain\AppData\Local\Temp\npm-12048-ff3f2333\git-cache-30786998\9c5a29ff037c28af07479eaa27591f1b4a0e5b66: error: cannot spawn sh: No such file or directory
npm ERR! git clone C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-90f8e8d5 C:\Users\Sylvain\AppData\Local\Temp\npm-12048-ff3f2333\git-cache-30786998\9c5a29ff037c28af07479eaa27591f1b4a0e5b66: fatal: unable to fork
npm ERR! git clone C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-90f8e8d5 C:\Users\Sylvain\AppData\Local\Temp\npm-12048-ff3f2333\git-cache-30786998\9c5a29ff037c28af07479eaa27591f1b4a0e5b66:
npm ERR! git fetch -a origin (https://github.com/ongoworks/node-authorize-net.git) fatal: Unable to find remote helper for 'https'
npm ERR! git clone --template=C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:ongoworks/node-authorize-net.git C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-e88b3c4b: Cloning into bare repository 'C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-e88b3c4b'...
npm ERR! git clone --template=C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:ongoworks/node-authorize-net.git C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-e88b3c4b: error: cannot spawn ssh: No such file or directory
npm ERR! git clone --template=C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:ongoworks/node-authorize-net.git C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-e88b3c4b: fatal: unable to fork
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Users\\Sylvain\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.1_1\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe" "C:\\Users\\Sylvain\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.1_1\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js" "i"
npm ERR! node v4.5.0
npm ERR! npm  v3.10.6
npm ERR! code 128

npm ERR! Command failed: git -c core.longpaths=true clone --template=C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:ongoworks/node-authorize-net.git C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-e88b3c4b
npm ERR! Cloning into bare repository 'C:\Users\Sylvain\AppData\Roaming\npm-cache\_git-remotes\git-github-com-ongoworks-node-authorize-net-git-e88b3c4b'...
npm ERR! error: cannot spawn ssh: No such file or directory
npm ERR! fatal: unable to fork
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     ....reaction\reaction\npm-debug.log
```

when i start reaction with reaction, i have:
```
Using settings file at settings/dev.settings.json

Setting up plugin imports...

[[[[[ C:\Users\Sylvain\Documents\App\Projects\ecommerce\reaction\reaction
]]]]]

=> Started proxy.
=> Started MongoDB.
```
then it hangs.

i'm on windows 10. i have installed all required tools. node/win32 SSL/MS build tools/Git/Imagemagick
```
reaction -v 
Node: 5.0.0
NPM: 3.9.0
Reaction: 0.16.1
Reaction CLI: 0.4.20
```

error as below: (Not in NPM registry) Shall wait and try after some time.

npm ERR! 404 Not found : @reactioncommerce/authorize-net
npm ERR! 404
npm ERR! 404  **'@reactioncommerce/authorize-net' is not in the npm registry.**
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'reaction'

`reaction init -b authorize-on-npm` removed the previous error. 

I have now an error with bcrypt, but reaction is working. It takes at least 6 mins to start but it could be linked to the bcrypt error, which i'm investigating further

thanks!


```
Using settings file at settings/dev.settings.json

Setting up plugin imports...

[[[[[ C:\Users\Sylvain\Documents\App\Projects\ecommerce\reaction\test4
]]]]]

=> Started proxy.
=> Started MongoDB.
cfs:gridfs: updating npm dependencies -- mongodb, gridfs-stream...
W20160930-12:54:08.962(8)? (STDERR) Note: you are using a pure-JavaScript implementation of bcrypt.
W20160930-12:54:09.123(8)? (STDERR) While this implementation will work correctly, it is known to be
W20160930-12:54:09.124(8)? (STDERR) approximately three times slower than the native implementation.
W20160930-12:54:09.125(8)? (STDERR) In order to use the native implementation instead, run
W20160930-12:54:09.125(8)? (STDERR)
W20160930-12:54:09.125(8)? (STDERR)   meteor npm install --save bcrypt
W20160930-12:54:09.126(8)? (STDERR)
W20160930-12:54:09.127(8)? (STDERR) in the root directory of your application.
I20160930-12:54:14.912(8)? 04:54:14.910Z  INFO Reaction: Load default data from /private/data/
I20160930-12:54:15.657(8)? 04:54:15.655Z  INFO Reaction: JobServer started
I20160930-12:54:15.677(8)? 04:54:15.661Z  WARN Reaction: Skipped loading settings from reaction.json.
I20160930-12:54:18.895(8)? 04:54:18.864Z  INFO Reaction: Starting createDefaultAdminUser
I20160930-12:54:18.982(8)? 04:54:18.979Z  INFO Reaction: Using meteor --settings to create admin user
I20160930-12:54:20.766(8)? 04:54:20.765Z  WARN Reaction:
```

when i do `meteor npm install --save bcrypt`
I get
 `> bcrypt@0.8.7 install C:\Users\Sylvain\Documents\App\Projects\ecommerce\reaction\test4\node_modules\bcrypt
> node-gyp rebuild

Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
TRACKER : error TRK0005: Failed to locate: "CL.exe". The system cannot find the file specified. [C:\Users\Sylvain\Documents\App\Projects\ecommerce\rea
ction\test4\node_modules\bcrypt\build\bcrypt_lib.vcxproj]


gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\Sylvain\AppData\Local\.meteor\packages\meteor-tool\1.4.1_1\mt-os.windows.x86_32\dev_bundle\lib\node_modules\node-gyp\lib\build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Windows_NT 10.0.14393
gyp ERR! command "C:\\Users\\Sylvain\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.1_1\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe" "C:\\Users\\Sylvain\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.1_1\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Sylvain\Documents\App\Projects\ecommerce\reaction\test4\node_modules\bcrypt
gyp ERR! node -v v4.5.0
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Users\\Sylvain\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.1_1\\mt-os.windows.x86_32\\dev_bundle\\bin\\node.exe" "C:\\Users\\Sylvain\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.4.1_1\\mt-os.windows.x86_32\\dev_bundle\\lib\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "bcrypt"
npm ERR! node v4.5.0
npm ERR! npm  v3.10.6
npm ERR! code ELIFECYCLE

npm ERR! bcrypt@0.8.7 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bcrypt@0.8.7 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the bcrypt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bcrypt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls bcrypt
npm ERR! There is likely additional logging output above.`

--- a/imports/plugins/included/authnet/server/methods/authnet.js
+++ b/imports/plugins/included/authnet/server/methods/authnet.js
@@ -6,7 +6,7 @@ import { Meteor } from "meteor/meteor";
 import { check, Match } from "meteor/check";
 import { Promise } from "meteor/promise";
 
-import AuthNetAPI from "authorize-net";
+import AuthNetAPI from "@reactioncommerce/authorize-net";
 import { Reaction, Logger } from "/server/api";
 import { Packages } from "/lib/collections";
 import { PaymentMethod } from "/lib/collections/schemas";

--- a/package.json
+++ b/package.json
@@ -19,8 +19,8 @@
     "url": "https://github.com/reactioncommerce/reaction/issues"
   },
   "dependencies": {
+    "@reactioncommerce/authorize-net": "^1.0.7",
     "accounting-js": "^1.1.1",
-    "authorize-net": "github:ongoworks/node-authorize-net",
     "autonumeric": "^1.9.45",
     "autosize": "^3.0.17",
     "avalara-taxrates": "^1.0.1",

